contracts/rosca.sol

```diff
@@ -70,8 +72,8 @@ contract rosca {
      */
     function rosca(
```

Comment:
do you think we can have comments above the contract and the methods? it can help with future code maintenance, and we'll need to add them anyway before security audit, ICO etc.